packages/flex-plugin-scripts/src/scripts/remove.ts

```diff
@@ -43,10 +43,14 @@ export const _doRemove = async () => {
   const pluginName = logger.colors.blue(paths.packageName);
   const credentials = await getCredential();
   const runtime = await _getRuntime(credentials);
+  if (!runtime.environment) {
+    throw new FlexPluginError('No Runtime environment was found');
+  }
+  const environment = runtime.environment;
```

Comment:
this var seems unnecessary

Reply:
Typescript stuff - it forgets that I've tested for this being defined inside the closure

Reply:
💩